po/common/common.pot

```diff
@@ -4845,23 +4845,3 @@ msgstr "High environmental impact"
 msgctxt "attribute_ecoscore_e_description_short"
 msgid "Very high environmental impact"
 msgstr "Very high environmental impact"
```

Comment:
```suggestion
msgstr "Very high environmental impact"

# keep the %s, it will be replaced by an allergen
msgctxt "contains_s"
msgid "Contains: %s"
msgstr "Contains: %s"

# keep the %s, it will be replaced by an allergen
msgctxt "may_contain_s"
msgid "May contain: %s"
msgstr "May contain: %s"

# keep the %s, it will be replaced by an allergen
msgctxt "does_not_contain_s"
msgid "Does not contain: %s"
msgstr "Does not contain: %s"

# keep the %s, it will be replaced by an allergen
msgctxt "without_s"
msgid "Without %s"
msgstr "Without %s"
```